### ISSUE TITLE
[3DS] Driv3r still keeps freezing on MGBA3DS(?)

### ISSUE THREAD

**Issue by qamichele59** [mgba-emu/mgba#2370]
It actually loads the game, but it always i got this Arm11 error before loading the city accidentally so any to fix this?

**Comment by qamichele5959** [comment]
Without it, this game will throw a "card removed" on bottom screen after intro screen as well.

**Comment by qamichele5959** [comment]
I tried all work on GBAR2 commit 20200812 on DSi console.

**Comment by qamichele5959** [comment]
In Sanrio My Melody new 3ds console, the game will show up into black screen as soon as it the intro screen is shown.

**Comment by endrift** [comment]
Do not post your complaints in unrelated threads. You've already been banned from the Discord for disruptive behavior so I will not hesitate to do the same here.

**Comment by endrift** [comment]
Also I need more information about how you're running the games--CIA or 3DSX, which version of mGBA, if you've got a custom firmware or not, etc

**Comment by qamichele5959** [comment]
Now all run on the older GBARunner2 builds but everything is so on the latest mGBA3DS builds :/

**Comment by endrift** [comment]
This issue now covers #2371, #2372, #2373, and #2374, as they all appear to be the same issue, which seems to be an issue with @qamichele5959's setup.

**Comment by endrift** [comment]
If you file another issue without giving me any of the information I have requested you will be blocked permanently. This is a chronic issue with you and why you got banned from Discord.

**Comment by qamichele59** [comment]
Unblocks discord mgba server?

**Comment by endrift** [comment]
You will not get unblocked if you continue to ignore any attempt at helping you.

**Comment by qamichele59** [comment]
[Does have this sugar sugar rune game work with center and mask mode enable on gbar2? How about all work on PC using mGBA?](https://github.com/Gericom/GBARunner2/issues/227)

**Comment by endrift** [comment]
If you do not give the information provide you are little more than a leech draining the resources of the people attempting to help you. You cannot ask for help and ignore attempts to get information that would help solve your issues or actual explanations.

**Comment by qamichele59** [comment]
Endrift unbans discord server mgba?

**Comment by endrift** [comment]
Goodbye.
